docs/resources/security_monitoring_default_rule.md

```diff
@@ -21,24 +24,30 @@ resource "datadog_security_monitoring_default_rule" "adefaultrule" {
 }
 ```
 
-## Argument Reference
+## Schema
+
+### Optional
+
+- **case** (Block List, Max: 5) Cases of the rule, this is used to update notifications. (see [below for nested schema](#nestedblock--case))
+- **enabled** (Boolean, Optional) Enable the rule.
+- **id** (String, Optional) The ID of this resource.
 
-The following arguments are supported:
+<a id="nestedblock--case"></a>
+### Nested Schema for `case`
 
--   `enabled`: (Optional, default = True) Whether the default rule is enabled.
--   `case`: (Optional) Change the notifications of a case.
-    -   `status`: Severity of the case.
-    -   `notifications`: Notification targets for the case.
+Required:
 
-## Importing
+- **notifications** (List of String, Required) Notification targets for each rule case.
+- **status** (String, Required) Status of the rule case to match.
 
-Default rules need to be imported using their ID before applying.
+## Import
 
-```hcl
+Import is supported using the following syntax:
+
+```shell
+# Default rules need to be imported using their ID before applying.
 resource "datadog_security_monitoring_default_rule" "adefaultrule" {
 }
-```
 
-```
 terraform import datadog_security_monitoring_default_rule.adefaultrule m0o-hto-lkb
```

Comment:
hmm this is not how it's presented in the other ones we migrated. Should we do something about the other ones ? (this seems more descriptive to me)

Reply:
I had a local build of tfplugindocs for the old ones. It was slightly modified to help me figure out most of the places we were missing descriptions so I could jumpstart this work. 

I upgraded tfplugindocs to the latest available before generating this one.
I do think it makes sense to do another pass of generation on the generation docs, but I want to finish getting the scaffolding (examples, first docs pass) done and just start doing full generation PRs with smaller diffs